src/main/java/nl/cherement/jak/controller/BoardController.java

```diff
@@ -47,9 +46,9 @@ public boolean cardMoved() {
         return true;
     }
 
-    @PostMapping("/user/{id}")
-    public HttpStatus addUser(@PathVariable("id") Long boardId, @RequestBody UserEntity userEntity) {
-        service.addUser(boardId, userEntity.getId());
+    @PostMapping("/user/{boardId}/{userId}")
+    public HttpStatus addUser(@PathVariable("boardId") Long boardId, @PathVariable("userId") Long userId) {
```

Comment:
Er is toch afgesproken dat als er iets in de controller word gepost. Dat de entity weer word teruggegooid

Reply:
Oh das waar ook. Hij zou eigenlijk de BoardEntity moeten returnen.